docker-compose.sov.yml

```
@@ -98,6 +98,23 @@ services:
     networks:
       - sov-network
 
+  evm-prover:
+    image: ghcr.io/celestiaorg/celestia-zkevm-ibc-demo/evm-prover:latest
+    container_name: evm-prover
+    environment:
+      # TENDERMINT_RPC_URL should be the SimApp which is acting as a substitute
+      # for Celestia (with IBC Eurekea enabled).
+      - TENDERMINT_RPC_URL=http://simapp-validator:26657
+      - RPC_URL=http://reth:8545
+      - EVM_PROTO_DESCRIPTOR_PATH=proto_descriptor.bin
+    ports:
+      - "50052:50052"
+    depends_on:
+      beacond:
+        condition: service_started
+    networks:
+      - rollkit-network
```

```suggestion
      - sov-network
```

we can also delete the evm-prover section from this file b/c we're going to delete the file in another PR soon anyway. See https://github.com/celestiaorg/celestia-zkevm-ibc-demo/issues/156